docs/reference/glossary.md

```diff
@@ -236,6 +248,7 @@ then there is a guarantee that T_N will never be included in the blockchain.
 ### Indexer
 
 - **[Indexer](../indexer/indexer-landing.md)** is the component of Aptos that retrieves, processes, and efficiently stores raw data in the database to provide speedy access to the Aptos blockchain state.
+- It provides a gRPC endpoint to submit GraphQL queries against as well as sample processor code to transform raw blockchain data.
```

Comment:
Technically, the gRPC endpoint is not the indexer.

This is yet another wordy and confusing one.

Indexer is a service that processes transactions and stores them in a more efficient way for specific use cases.  Indexer often refers to the database, the transaction processor, or both.